src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const DAY_OF_RENT = days * 40;
```

Comment:
and actually, it's not the constant, 'cause this var depending on the days argument

Reply:
let's rename it in camelCase